src/components/dapp-staking/modals/ModalDappDetails.vue

```diff
@@ -1,41 +1,143 @@
 <template>
   <Modal>
     <template #content>
-      <div class="tw-flex tw-flex-col tw-justify-center tw-items-center dark:tw-text-darkGray-100">
-        <Avatar :url="dapp.iconUrl" class="tw-w-36 tw-h-36" />
-        <div class="tw-my-8 tw-text-2xl tw-font-semibold">{{ dapp.name }}</div>
-        <div>{{ dapp.description }}</div>
-        <div class="tw-my-8 tw-w-full tw-text-lg">
-          <a :href="dapp.url" target="_blank">{{ dapp.url }}</a>
+      <div class="tw-flex tw-flex-row tw-flex-wrap">
+        <q-carousel
+          v-model="slide"
+          class="tw-w-96 rounded-borders"
+          swipeable
+          animated
+          navigation
+          arrows
+          infinite
+        >
+          <q-carousel-slide name="video">
```

Comment:
![Screen Shot 2021-12-14 at 4 06 45 PM](https://user-images.githubusercontent.com/40356749/146024269-15bd8524-a544-4ef5-9fe6-f47a7278d21c.png)
Can we make the modal a bit longer to the sides so we don't have the empty space under the media carousel?

Reply:
I agree. I will check which of the proposed solutions looks better